### ISSUE TITLE
TypeError: "strict" property must be of type boolean, got string

### ISSUE THREAD

**Issue by mybearworld** [face-hh/yappacino#2]
I get this error when trying to `bun src/backup/yapped_index.js`:
```
375 |
376 | KEYWORDS["]:"] = KEYWORDS[":"];
377 | KEYWORDS["}:"] = KEYWORDS[":"];
378 |
379 | async function main() {
380 |     const { values } = parseArgs({
                             ^
TypeError: "strict" property must be of type boolean, got string
 code: "ERR_INVALID_ARG_TYPE"

      at ...\yappacino\src\backup\yapped_index.js:380:24
      at main (...\yappacino\src\backup\yapped_index.js:379:20)
      at ...\yappacino\src\backup\yapped_index.js:490:1
```

**Comment by face-hh** [comment]
That's really weird, does the same happen if you run the executable from the Release tab instead?

**Comment by mybearworld** [comment]
I'm on Windows, so the executable doesn't work

**Comment by face-hh** [comment]
Alright, could you try `bun build src/backup/yapped_index.js --compile --outfile=yap && ./yap`?

**Comment by mybearworld** [comment]
Still errors:

```
58 |   }
59 |   line = line.replace(REGEX.types, ")");
60 |   return { ARGS, no_types_line: line };
61 | };
62 | async function main() {
63 |   const { values } = parseArgs({
    args: Bun.argv,
    options: {
      shut_the_fuck_up: {
        type: "boolean"
      }
    },
    strict: __BOOLEANS__.FALSE,
    allowPositionals: __BOOLEANS__.TRUE
  });
                          ^
TypeError: "strict" property must be of type boolean, got string
 code: "ERR_INVALID_ARG_TYPE"

      at B:/~BUN/root/yap:63:22
      at main (B:/~BUN/root/yap:62:21)
      at B:/~BUN/root/yap:368:1
```

`__BOOLEANS__.FALSE` is a string, not the boolean value. https://github.com/face-hh/yappacino/blob/a6c542d5d22c60b468dda7e52a0247359049a79f/src/index.yap#L10

**Comment by face-hh** [comment]
Dang that's... I'll try working on a fix tomorrow.

I made `true` & `false` replace to `__BOLEANS__.TRUE` and `__BOOLEANS__.FALSE`  for some reason...

Until then you can probably modify your `yapped_index` and replace `__BOOLEANS__.TRUE` and `__BOOLEANS.FALSE` with `true` and `false` respectively

**Comment by face-hh** [comment]
Fixed in v1.0.1
